config/prow/plugins.yaml

```diff
@@ -1166,6 +1166,15 @@ plugins:
     - wip
     - yuks
 
+  etcd:
+    plugins:
+    - welcome
+    - assign
+    - trigger
+    - wip
+    - hold
+    - owners-label
```

Comment:
This is we really haved so i put they here and `owners-label` is a new plugin for auto add label. Maybe we also want to add the plugin for `lgtm` and `approve`

```yaml
    - welcome
    - assign
    - trigger
    - wip
```